src/cocotb/share/lib/vhpi/VhpiImpl.cpp

```diff
@@ -867,72 +867,105 @@ GpiObjHdl *VhpiImpl::native_check_create(int32_t index, GpiObjHdl *parent) {
 
 GpiObjHdl *VhpiImpl::get_root_handle(const char *name) {
     vhpiHandleT root = NULL;
+    std::string root_name;
+
```

Comment:
I know the existing style is to declare everything at the top here, but let's make a change. This style is like 30 years antiquated.